packages/clerk-js/src/ui/elements/InputWithIcon.tsx

```diff
@@ -1,25 +1,30 @@
 import React from 'react';
 
 import { Flex, Input } from '../customizables';
-import type { PropsOfComponent } from '../styledSystem';
+import type { PropsOfComponent, ThemableCssProp } from '../styledSystem';
 
-type InputWithIcon = PropsOfComponent<typeof Input> & { leftIcon?: React.ReactElement };
+type InputWithIcon = PropsOfComponent<typeof Input> & { leftIcon?: React.ReactElement } & {
+  containerSx?: ThemableCssProp;
+};
 
 export const InputWithIcon = React.forwardRef<HTMLInputElement, InputWithIcon>((props, ref) => {
-  const { leftIcon, sx, ...rest } = props;
+  const { leftIcon, sx, containerSx, ...rest } = props;
   return (
     <Flex
       center
-      sx={theme => ({
-        width: '100%',
-        position: 'relative',
-        '& .cl-internal-icon': {
-          position: 'absolute',
-          left: theme.space.$4,
-          width: theme.sizes.$3x5,
-          height: theme.sizes.$3x5,
+      sx={theme => [
+        {
+          width: '100%',
+          position: 'relative',
+          '& .cl-internal-icon': {
+            position: 'absolute',
+            left: theme.space.$4,
+            width: theme.sizes.$3x5,
+            height: theme.sizes.$3x5,
```

Comment:
```suggestion
            height: theme.sizes.$3x5,
            pointerEvents: 'none'
```

Make it so that we can click anywhere within the input and focus. Currently blocked when clicking icon directly.

Reply:
Thanks for catching it! Updated here https://github.com/clerk/javascript/pull/4942/commits/5adcc9ffe5e0aa6e90fe40a83bad35464fd4629f